server/config/type.js

```diff
@@ -69,7 +69,9 @@ const ConfigType = class ConfigType {
         });
 
         await Promise.all(relations.map(async (relation) => {
-          await strapi.entityService.delete(this.queryString, relation.id);
+          await strapi.query(queryString).delete({
```

Comment:
Amazing, this indeed seems to fix the integration tests.
What do you reckon was the fix?

To me, looking at it, I think prior to this commit you used the wrong query string.
You changed `this.queryString` to `queryString`. I believe that was the fix.
Meaning you should still be able to get away with a `entityService.delete` instead of reverting back to using the `strapi.query` layer.